src/renderer/entities/wallet/lib/account-utils.ts

```diff
@@ -70,7 +72,7 @@ function isWatchOnlyAccount(account: Partial<AnyAccount>): account is WatchOnlyA
   );
 }
 
-function isVaultBaseAccount(account: Partial<AnyAccount>): account is VaultBaseAccount {
+function isBaseAccount(account: Partial<AnyAccount>): account is BaseAccount {
```

Comment:
Why did you changed name of interface to `BaseAccount`? It exists only in PV

Reply:
> PV contains only 2 types of accounts - VaultChainAccount & VaultShardAccount
BaseAccount appears only in WO & SingleShard

So there is no point in saving `Vault` prefix

Reply:
Without Vault prefix it harder to get context of this account. Also there is a huge affected area because of this small change. Let's roll it back